webui/conf/browsersync.conf.js

```diff
@@ -1,11 +1,26 @@
 const conf = require('./gulp.conf');
+const proxy = require('http-proxy-middleware');
```

Comment:
This dependency allows you to run in dev mode with a real Traefik backend running on localhost:8080.
For instance, when you run :
`docker run -p 8080:8080 -p 80:80 -v /var/run/docker.sock:/var/run/docker.sock  traefik --debug --web --docker --docker.watch`
and then
`npm run serve`
On http://localhost:3000/ , the server is now able to connect to the running traefik /api and /health endpoints.
I think it could be useful for testing with a real server, and since it is in dev dependencies, it is not bundled in the final release. Let me know if you want me to remove this.